yarn.lock

```diff
@@ -6789,12 +6789,12 @@ color-convert@^2.0.1:
   dependencies:
     color-name "~1.1.4"
 
-color-name@1.1.3, color-name@^1.0.0:
+color-name@1.1.3:
   version "1.1.3"
   resolved "https://registry.yarnpkg.com/color-name/-/color-name-1.1.3.tgz#a7d0558bd89c42f795dd42328f740831ca53bc25"
   integrity sha1-p9BVi9icQveV3UIyj3QIMcpTvCU=
 
-color-name@^1.1.4, color-name@~1.1.4:
+color-name@^1.0.0, color-name@^1.1.4, color-name@~1.1.4:
```

Comment:
So.... I don't totally get if we're de-duping this library why we have two entries still?

Reply:
The deduplication run I did is for "accidental" deduplication: when we specify to compatible versions ranges (in this case three: color-name@^1.0.0, color-name@^1.1.4 and color-name@~1.1.4) that they should have been just one package from the get-go, but because yarn/npm oddities they aren't.

This doesn't change deliberate duplications, when we depend on two incompatible ranges. We can review those later, but they are trickier to change.

Reply:
Thanks for the clarification!